src/ledctl/ledctl.c

```diff
@@ -1050,6 +1040,34 @@ static led_status_t _init_ledctl_conf(void)
 	return ledmon_init_conf(&conf, LED_LOG_LEVEL_WARNING, LEDCTL_DEF_LOG_FILE);
 }
 
+static const char *getLogLevelName(enum led_log_level_enum log_level)
+{
+	switch (log_level) {
+	case LED_LOG_LEVEL_UNDEF:
+		return "UNDEF";
+	case LED_LOG_LEVEL_QUIET:
+		return "QUIET";
+	case LED_LOG_LEVEL_ERROR:
+		return "ERROR";
+	case LED_LOG_LEVEL_WARNING:
+		return "WARNING";
+	case LED_LOG_LEVEL_INFO:
+		return "INFO";
+	case LED_LOG_LEVEL_DEBUG:
+		return "DEBUG";
+	case LED_LOG_LEVEL_ALL:
+		return "ALL";
+	default:
+		return "UNDEF";
+	}
+}
+
+static void print_configuration(void)
+{
+	printf("Log level: %s\n", getLogLevelName(conf.log_level));
```

Comment:
this one in made to be parser frendly, maybe not big deal but I would prefer to avoid spaces.
LOG_LEVEL=level
LOG_PATH=/ddfs/fdfs